model/transaction.go

```diff
@@ -15,38 +15,33 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package transaction
+package model
 
 import (
 	"context"
 	"time"
 
-	"github.com/elastic/apm-server/model/span"
-
 	"github.com/elastic/beats/v7/libbeat/beat"
 	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/libbeat/monitoring"
 
-	m "github.com/elastic/apm-server/model"
-	"github.com/elastic/apm-server/model/metadata"
 	"github.com/elastic/apm-server/transform"
 	"github.com/elastic/apm-server/utility"
 )
 
 const (
-	processorName      = "transaction"
-	transactionDocType = "transaction"
-	emptyString        = ""
+	transactionProcessorName = "transaction"
+	transactionDocType       = "transaction"
 )
 
 var (
-	Metrics         = monitoring.Default.NewRegistry("apm-server.processor.transaction")
-	transformations = monitoring.NewInt(Metrics, "transformations")
-	processorEntry  = common.MapStr{"name": processorName, "event": transactionDocType}
+	TransactionMetrics         = monitoring.Default.NewRegistry("apm-server.processor.transaction")
```

Comment:
I recognize this is just maintaining the current visibility but can these (`ErrorMetrics`, `SpanMetrics`, etc) not be exported now?

Reply:
Done for all but SourcemapMetrics, which is used by processor/asset. I've left a TODO for that.